requirements/build.txt

```diff
@@ -7,3 +7,4 @@ setuptools-scm>=8
 torch==2.7.0
 wheel
 jinja2>=3.1.6
+pathspec
```

Comment:
I don't think this is necessary? `pre-commit` will create a fresh environment for this hook and you have already specified that this environment should contain `pathspec`

```suggestion
```

Reply:
Yeah, you're right. I think I added this when I was running the script manually, before adding to pre-commit.